src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java

```diff
@@ -21,82 +21,71 @@
 import com.scad.expensestracker.models.ExpensesTrackerItem;
 import com.scad.expensestracker.repositories.ExpensesTrackerRepository;
 
-import ch.qos.logback.core.model.Model;
 import jakarta.validation.Valid;
 
 @Controller
 public class ExpensesTrackerItemController {
     private final Logger logger = LoggerFactory.getLogger(ExpensesTrackerItemController.class);
-   
-    @Autowired // This allows Spring to manage the lifecycle of the repository and ensures that 
-    // it's properly initialized when it's used in your controller.
-    private ExpensesTrackerRepository ExpensesTrackerRepository;
+
+    @Autowired
+    private ExpensesTrackerRepository expensesTrackerRepository;
 
     @GetMapping("/")
-    public ModelAndView index(Model model){
+    public ModelAndView index() {
         logger.debug("root to GET index");
         ModelAndView modelAndView = new ModelAndView("index");
-        List<ExpensesTrackerItem> expensesTrackerItems = new ArrayList<>();
-        ExpensesTrackerRepository.findAll().forEach(expensesTrackerItems::add);
+        List<ExpensesTrackerItem> expensesTrackerItems = (List<ExpensesTrackerItem>) expensesTrackerRepository.findAll();
         double totalPrice = expensesTrackerItems.stream().mapToDouble(ExpensesTrackerItem::getPrice).sum();
         modelAndView.addObject("ExpensesTrackerItems", expensesTrackerItems);
         modelAndView.addObject("today", Instant.now().atZone(ZoneId.systemDefault()).toLocalDate().getDayOfWeek());
         modelAndView.addObject("totalPrice", totalPrice);
-       
-
-        double averageDailyExpense = calculateAverageDailyExpense();
-        String dominantCategory = findDominantCategory();
-        modelAndView.addObject("averageDailyExpenses", averageDailyExpense);
-        modelAndView.addObject("mostExpensiveCategory", dominantCategory);
-
-
+        modelAndView.addObject("averageDailyExpenses", calculateAverageDailyExpense());
+        modelAndView.addObject("mostExpensiveCategory", findDominantCategory());
         return modelAndView;
     }
-    
+
     @PostMapping("/ExpensesTracker")
-    public String createExpensesTrackerItem(@Valid ExpensesTrackerItem expensesTrackerItem, BindingResult result, Model model) {
+    public String createExpensesTrackerItem(@Valid ExpensesTrackerItem expensesTrackerItem, BindingResult result)
+    {
         if (result.hasErrors()) {
             return "add-ExpensesTracker-item";
         }
         expensesTrackerItem.setCreatedDate(Instant.now());
         expensesTrackerItem.setModifiedDate(Instant.now());
-        ExpensesTrackerRepository.save(expensesTrackerItem);
+        expensesTrackerRepository.save(expensesTrackerItem);
```

Comment:
Why e is small? Is it working?

Reply:
its a mistake , i added it but then due to some issues of git push and git pull. it got mixed up